### ISSUE TITLE
How to get feature value in custom loss function

### ISSUE THREAD

**Issue by Gynshu** [tensorflow/tensorflow#50081]
How can i get not only y_true and y_pred, but the last feature value from row that predicts y_pred in custom loss function to calculate loss with it. I use rnn in tf.keras.

**Comment by jvishnuvardhan** [comment]
@Gynshu Can you please create a simple standalone code to reproduce the issue? A simple code example to demonstrate what you are trying to achieve in custom loss function. Thanks!
